crates/config/src/config.rs

```diff
@@ -229,173 +225,187 @@ impl TestNodeConfig {
+                fd.fee_model_config_v2.max_gas_per_batch.to_string().green(),
                 fd.fee_model_config_v2
                     .max_pubdata_per_batch
                     .to_string()
                     .green()
             );
-            sh_println!("\n");
         } else {
-            sh_println!("Network Configuration");
-            sh_println!("========================");
             sh_println!(
-                "Chain ID: {}",
+                r#"
+Network Configuration
+========================
+Chain ID: {}
+"#,
                 self.chain_id
                     .unwrap_or(TEST_NODE_NETWORK_ID)
                     .to_string()
                     .green()
             );
-            sh_println!("\n");
         }
-        sh_println!("Gas Configuration");
-        sh_println!("========================");
-        sh_println!(
-            "L1 Gas Price (gwei):               {}",
-            format_gwei(self.get_l1_gas_price().into()).green()
-        );
-        sh_println!(
-            "L2 Gas Price (gwei):               {}",
-            format_gwei(self.get_l2_gas_price().into()).green()
-        );
-        sh_println!(
-            "L1 Pubdata Price (gwei):           {}",
-            format_gwei(self.get_l1_pubdata_price().into()).green()
-        );
-        sh_println!(
-            "Estimated Gas Price Scale Factor:  {}",
-            self.get_price_scale().to_string().green()
-        );
+
+        // Gas Configuration.
         sh_println!(
-            "Estimated Gas Limit Scale Factor:  {}",
+            r#"
+Gas Configuration
+========================
+L1 Gas Price (gwei):               {}
+L2 Gas Price (gwei):               {}
+L1 Pubdata Price (gwei):           {}
+Estimated Gas Price Scale Factor:  {}
+Estimated Gas Limit Scale Factor:  {}
+"#,
+            format_gwei(self.get_l1_gas_price().into()).green(),
+            format_gwei(self.get_l2_gas_price().into()).green(),
+            format_gwei(self.get_l1_pubdata_price().into()).green(),
+            self.get_price_scale().to_string().green(),
             self.get_gas_limit_scale().to_string().green()
         );
-        sh_println!("\n");
 
-        sh_println!("Genesis Timestamp");
-        sh_println!("========================");
-        sh_println!("{}", self.get_genesis_timestamp().to_string().green());
-        sh_println!("\n");
+        // Genesis Timestamp.
+        sh_println!(
+            r#"
+Genesis Timestamp
+========================
+{}
+"#,
+            self.get_genesis_timestamp().to_string().green()
+        );
 
-        sh_println!("Node Configuration");
-        sh_println!("========================");
-        sh_println!("Port:               {}", self.port);
+        // Node Configuration.
         sh_println!(
-            "EVM Emulator:       {}",
+            r#"
+Node Configuration
+========================
+Port:               {}
+EVM Emulator:       {}
+Health Check Endpoint: {}
+ZK OS:              {}
```

Comment:
This table has misaligned fields. Migration to raw strings is already paying off dividends!